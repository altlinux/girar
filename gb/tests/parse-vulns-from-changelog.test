- FIXES
- Fixes:
  + CVE-2017-5376: Use-after-free in XSL
- (fixes CVE-1111-1111, BBBBBB, OVE-11111111-1111 BDU:1111-11111, MFSA-1111-11)
- fixes  
  CVE-2017-5380: Potential use-after-free during DOM manipulations
  + (fixes OVE-99999999-9999)
  + CVE-2017-9999
- (fIxEs CVE-2222-2222 XXXX , YYYY and CVE-2222-2222)

- Updated to 2.6.32-642.13.1.el6 (fixes CVE-3333-3333,
  , ,

  CVE-3333-3333).

(fixes MFSA-5555-55) (fixes
BDU:5555-55555)

- fixes
  + (fixes CVE-4444-4444 XXXX
