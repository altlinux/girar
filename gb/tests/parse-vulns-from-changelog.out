CVE-2017-5376
CVE-1111-1111
OVE-11111111-1111
BDU:1111-11111
MFSA-1111-11
CVE-2017-5380
OVE-99999999-9999
CVE-2017-9999
CVE-2222-2222
CVE-2222-2222
CVE-3333-3333
CVE-3333-3333
MFSA-5555-55
BDU:5555-55555
CVE-4444-4444
